SEImplementation/src/lib/Plugin/FlexibleModelFitting/FlexibleModelFittingTask.cpp

```diff
@@ -104,6 +104,14 @@ void printLevmarInfo(std::array<double, 10> info) {
 
 }
 
+static void updateFlags(Flags& flags, unsigned stop_reason) {
+  switch (stop_reason) {
```

Comment:
I have noticed the code for the GSL integration will return 0 as stop reason in case of unexpected result as well as a few levmar-compatible codes, maybe it should also be added to the possible errors. Or maybe this should reverse logic, expected stop reasons are ok everything else is considered an error?

Reply:
True. Another option would be to have a Flag on the `LeastSquareSummary` struct and let the engine-specific implementation decide?

Reply:
Yes, I think it's best to be flexible for future engine implementations